ceph_installer/templates.py

```diff
@@ -19,6 +19,10 @@
 fi
 
 
+echo "--> Installing Python for Ansible"
+sudo apt-get update
+sudo DEBIAN_FRONTEND=noninteractive apt-get -y install python
```

Comment:
ideally we would want this for Xenial only, but it certainly doesn't hurt because it is still valid in older Ubuntu versions